src/main/kotlin/kscript/app/AppHelpers.kt

```diff
@@ -255,16 +262,16 @@ $stringifiedRepos
 }
```

Comment:
Not directly related to this PR but I noticed the `build.gradle` file includes `mavenLocal()` by default. Is that intended ?